src/features/languages/generated/client/contentful-keys-to-migrate.json

```diff
@@ -20,6 +20,8 @@
   "constants.archived_declaration",
   "duplicates.warning",
   "error.passwordSubmissionError",
+  "location.jurisdictionType.UNION",
+  "location.jurisdictionType.DIVISION",
```

Comment:
Why do we need to add these Bangladesh requirements back in?  It is technical debt @sadmananik

Reply:
Yeah we don't need them in farajaland and can be removed from core as well after refactoring the unit tests

Reply:
These were generated auto while commiting.